config/swagger.yaml

```diff
@@ -767,6 +767,16 @@ paths:
           description: 'Hash of the block to fetch'
           required: true
           type : string 
+        - in: query
+          name: tx_types
+          description: 'Transactions types to show. Comma separated'
+          required: false
+          type : string 
+        - in: query
+          name: exclude_tx_types
+          description: 'Transactions types not to show. Comma separated'
```

Comment:
So according to the filter implementation above, `exclude_tx_types` takes precedence over `tx_types`, i.e. if a type appears in both, it's excluded. Should this be documented, perhaps?

Reply:
You're right, appending it